CONTRIBUTING.md

```diff
@@ -690,6 +690,45 @@ query localConnections {
 curl -s -H "Content-Type:application/json" -H "Authorization: Bearer ${DTX_ACCESS_TOKEN}" http://localhost:26636/gateway/v1/graphql/schema.graphql
 ```
 
+### Invoking the Control Plane Proxy API
+
+The Control Plane Proxy API allows you to make calls to the Confluent Cloud control plane, specifically for artifact and compute pool management endpoints defined by the configuration:
+
+```yaml
+ccloud-api-control-plane-regex: "(/artifact.*)|(/fcpm/v2/compute-pools.*)"
+```
```

Comment:
Should we add a link to the application.yml so that folks can figure out the full config path, `ide-sidecar.connections.ccloud.control-plane-base-url`?